android/build.gradle

```diff
@@ -19,30 +19,35 @@ group 'com.nimroddayan.flutternsd'
 version '1.0-SNAPSHOT'
 
 buildscript {
-    ext.kotlin_version = '1.5.20'
+    ext.kotlin_version = '1.7.10'
     repositories {
         google()
-        jcenter()
+        mavenCentral()
     }
 
     dependencies {
-        classpath 'com.android.tools.build:gradle:4.0.1'
+        classpath 'com.android.tools.build:gradle:7.3.1'
```

Comment:
PR title says AGP 8 but I see 7 here :)

Reply:
AGP 8 compatibility is for __consumer__ projects using AGP 8.
Here changing to 7 was not necessary for AGP 8 compatibility, but just part of general upgrades from ancient versions used in this repo.